pkg/broker/ingress/handler_test.go

```diff
@@ -59,7 +59,7 @@ const (
 
 	traceID = "4bf92f3577b34da6a3ce929d0e0e4736"
 
-	eventType = "test-event-type"
+	eventType = "com.google.cloud.pubsub.topic.publish"
```

Comment:
all these values will change with @yolocs PR https://github.com/google/knative-gcp/pull/1340

Reply:
Switched it to those newer ones, and now only accept event types matching "google.cloud" instead of "com.google.cloud"